framework/addons/data/addons.json

```diff
@@ -150,6 +150,36 @@
                 "text": "Dataverse content will be registered, but version history will not be copied to the registration."
             }
         },
+        "Dryad": {
+            "Permissions": {
+                "status": "partial",
+                "text": "Making an OSF project public or private is independent of making a Dryad package public or private. The OSF allows you to utlize the archived unembargoed version of Dryad Packages."
+            },
+            "View / download file versions": {
+                "status": "partial",
+                "text": "Files from the latest release of the selected Dryad package can be viewed/downloaded."
+            },
+            "Add / update files": {
+                "status": "NA",
+                "text": "Adding/updating files in the project via OSF is currently not allowed."
+            },
+            "Delete files": {
+                "status": "NA",
+                "text": "Files deleted via OSF will not be deleted in Dryad."
```

Comment:
Yup. The next release will feature pushing and pulling to Dryads staging and review databases, but for now, the only portions accessible will be the parts of the archive not under embargo.